### ISSUE TITLE
How to report an issue related to the Rakudo webpages?

### ISSUE THREAD

**Issue by bazzaar** [Raku/problem-solving#397]
I would like to see @nxadm's rakudo-pkg's placed at the top of the installation options for Linux users in the "Downloads" web page at the Rakudo website. The packages for all the major distributions are bang up-to-date, and work well as automatic system updates with just the simple addition of the 3rd-party repo.

However when I tried to add an issue using the link to Rakudo github issue tracker, I'm faced with a open issue list of over 1300 of which very few address problems with the webpages, and a tag list of 116 spread over 4 pages.

Even in this repo [Problem solving] it wasn't clear to me how to report a problem with the Rakudo webpages.

**Comment by coke** [comment]
when on rakudo.org, you can click on "Issue Trackers" in the main menu, and one of the options is "This website" - I'm guessing you clicked on the github issue tracker at the top. From a process perspective, even if you open a ticket there, an admin can transfer it to the website repo.

**Comment by raiph** [comment]
The page verbiage is wrong. I'll create a PR.

**Comment by nxadm** [comment]
> I would like to see @nxadm's rakudo-pkg's placed at the top of the installation options for Linux users in the "Downloads" web page at the Rakudo website

Thank you for the vote of confidence.

Without going deeper into the proposal above, I think the Download page illustrates the lack of focus @bazzaar [mentioned in an issue](https://github.com/Raku/Raku-Steering-Council/issues/56).

**Comment by raiph** [comment]
> when on rakudo.org, you can click on "Issue Trackers" in the main menu, and one of the options is "This website"

Right. But I just went through the same process as I imagined @bazzaar went through, and found myself treading the same footsteps. 

The initial verbiage is:

> Issue Trackers

.oO ( OK. Plural. Cool. )

> Our GitHub Issue Tracker is the main place to report problems with Rakudo or to submit possible feature suggestions. You can use it even if you're unsure where to file your report.

.oO ( OK. Singular. Presumably the other issue trackers aren't GitHub. I prefer RT, so let's read on. )

> RT Issue Tracker - these tickets have been migrated to github issues

.oO( Ah. Sounds like they use 3 or more issue tracker systems! Anyhow, I prefer GitHub so I'll stick with the first link. )

[ clicks original main GitHub link ]

.oO( New issue. Hmm. But maybe I should search first? What a basic mistake to take me here! Someone should open an issue? [ like rakudo/rakudo.org#93 ;) ]  )

[ clicks to see list ]

.oO( Woah. 1300 issues! My goodness that isn't a good sign. Maybe I should just go away? )

.oO( Maybe they've got 1300 because people keep duplicating them because they're taken to the new issue screen? I should file an issue about that. That'd be a yak shave, but still, I wanna be helpful... )

.oO( What to search for to see existing rakudo.org website issues? Let's try searching for open issues with `website` in them. Loads of matches. But they're clearly just matches for the word "website" in them, not to do do with website issues. `rakudo.org`? Grr. Same problem. )

.oO( Ah, what about labels! That'll be it. Woah. 116 labels! What label do I use for the yak shave (remove the `/new`) under? Or my original problem (issue with website)? Oh well, let's try searching the labels. Let's deal with my original problem first. Hmm. `web`? Nope. `website`? Nope. `rakudo.org`? Ah, 5 matches! Hmm. `JVM`?!? Hmm. Let's click on it. Huh? Grr. Same problem --they're all nothing to do with rakudo.org per se!  )

.oO( I know. I heard something about a problem-solving repo. I'll go file there... )

**Comment by jubilatious1** [comment]
I guess __Docs__ issues go on the `docs` pages? 

https://github.com/Raku/doc

And __Rakudo__ issues on the `rakudo` pages?

https://github.com/rakudo/rakudo

_(...not sure about the Rakudo website)._

And Community Modules on the `ecosystem` pages?

https://github.com/Raku/ecosystem

_(...wasn't this once called `Raku Community Modules`?)._

And finally if you can't figure out an appropriate Repo, file an Issue at `problem-solving`:

https://github.com/Raku/problem-solving
